plugins/statuslines/lualine.nix

```
@@ -1,260 +1,281 @@
 {
   lib,
-  helpers,
-  config,
   pkgs,
   ...
 }:
-with lib;
 let
-  cfg = config.plugins.lualine;
+  inherit (lib.nixvim) defaultNullOpts mkSettingsRenamedOptionModules;
+  inherit (lib) types mkOption;
 
+  # TODO: do we really need this
   mkSeparatorsOption =
     {
       leftDefault ? " ",
       rightDefault ? " ",
```

```suggestion
      leftDefault,
      rightDefault,
```

These defaults are only used once, let's just be explicit in all usage?